doc/api/worker_threads.md

```diff
@@ -61,6 +61,26 @@ Worker threads inherit non-process-specific options by default. Refer to
 [`Worker constructor options`][] to know how to customize worker thread options,
 specifically `argv` and `execArgv` options.
 
+## `worker.connect(target[, data][, timeout])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `target` {number} The target thread id.
+* `data` {any} Any arbitrary, cloneable JavaScript value.
+* `timeout` {number} Time to wait in milliseconds. By default it's `undefined`, which
+  means wait forever.
+* Returns: {Promise} A promise for a `MessagePort`.
+
+Establishes a connection to another worker thread in the same process, returning a
+`MessagePort` that can be used for the communication.
+
+The target thread must have a connection listener setup via [`worker.setConnectionsListener()`][]
+otherwise the connection request will fail.
```

Comment:
This could use an example. If this is `const worker = new Worker(...); worker.connect(...)` why is the target necessary? Where does the target id come from?

Reply:
I just added an example. Does it look better?